docs/symphony-book/scenarios/helm-chart-image-in-privateCR.md

```diff
@@ -0,0 +1,81 @@
+# Scenario: using private container registry in helm chart
+In symphony context, instead of creating the iamge pull secret in the helm chart and passing the key and password in plain-text, you can pass the password using $secret to the chart values.
+
+You can create a solution file like this:
+The $secret refers to the kubenetes secret. $secret("SECRETNAME", "FIELDNAME")
```

Comment:
k8s secret in which namespace? (same namespace?)

Reply:
Maybe we can organize the steps with explicit bullet points for easier understanding. 
0. Create a helm chart and expose image pull secret via helm values.
1. Store secrets in k8s secret store.
    a. Create it manually.
    b. Sync with Azure Arc Secret Sync manager if it is arc-extension.
2. Create solutioncontainer/solution and use $secret to pass values.
3. Create target
4. Create instance. Take a screenshot to guide users to check their images have been pulled from container registry.